src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.ML.Runtime.Internal.CpuMath
 {
     public static partial class CpuMathUtils
     {
-        public static void MatrixTimesSource(bool transpose, ReadOnlySpan<float> matrix, ReadOnlySpan<float> source, Span<float> destination, int stride)
+        public static void MatrixTimesSource(bool transpose, float[] matrix, ReadOnlySpan<float> source, Span<float> destination, int stride)
```

Comment:
This should stay a Span. Don't change it back to an array.

Reply:
I earlier tried with ROS but some tests are not then finding this function and I am getting Missing Method Exception. I will try to find more info on the reason behind it or source of the error